core/assets/bundles/bundle.properties

```diff
@@ -1303,6 +1304,7 @@ block.swarmer.description = A medium-sized missile turret. Attacks both air and
 block.salvo.description = A larger, more advanced version of the Duo turret. Fires quick salvos of bullets at the enemy.
 block.fuse.description = A large, close-range shrapnel turret. Fires three piercing blasts at nearby enemies.
 block.ripple.description = An extremely powerful artillery turret. Shoots clusters of shells at enemies over long distances.
+block.clapper.description = A powerful turret, shoots large plastane bombs, handy against large clusters of weak enemies.
```

Comment:
Do you mean "...large plastanium bombs; handy against..."?

Reply:
sorry, my knowledge of English is limited to google translator